apps/server/src/routes/agent/index.ts

```
@@ -51,6 +51,7 @@ import { processToolCalls } from './utils';
 import { env } from 'cloudflare:workers';
 import type { Connection } from 'agents';
 import { openai } from '@ai-sdk/openai';
+import { openai } from '@ai-sdk/openai';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'openai'. Consider to delete it or rename it.

'openai' is defined here:


(lint/suspicious/noRedeclare)